common/djangoapps/student/views.py

```diff
@@ -693,11 +707,13 @@ def dashboard(request):
         'order_history_list': order_history_list,
         'courses_requirements_not_met': courses_requirements_not_met,
         'nav_hidden': True,
+        'course_programs': course_programs,
+        'program_marketing_url': urljoin(settings.MKTG_URLS.get('ROOT'), 'xseries' + '/marketing')
```

Comment:
actually since this is being set for all courses/programs, it should not have the marketing slug - that should get appended when we're handling xseries display for individual course runs.